Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -51,7 +51,7 @@ def create_indicators_from_response(self):
                     continue
                 published_iso = published.strftime('%Y-%m-%dT%H:%M:%S')
                 publications.append({
-                    'timestamp': indicator.get('published'),
+                    'timestamp': published_iso,
```

Comment:
why was this changed?

Reply:
Because there are instances where the 'published` is in a format that the server doesn't support and raises the following error:
```
Error from create indicators response: [400] Failed with error: [1:1213] failed to parse field [CustomFields.publications_grid.timestamp_date] of type [date] in document with id '2d6667d68d3028bd682e2a5a19b792aa'. Preview of field's value: 'Fri, 8 Mar 2019 19:45:00 +0000'. Other reasons: [[{illegal_argument_exception failed to parse date field [Fri, 8 Mar 2019 19:45:00 +0000] with format [basic_date_time||strict_date_optional_time||epoch_millis||yyyy-MM-dd HH:mm:ssz||strict_date_optional_time_nanos||yyyy-MM-dd HH:mm:ss.SSSSSSz||yyyy-MM-dd HH:mm:ss]}]]
```